scripts/userstyles.yml

```
@@ -655,6 +657,14 @@ userstyles:
     readme:
       app-link: "https://www.npmjs.com/"
     current-maintainers: [*uncenter]
+  ollama:
+    name: Ollama
+    categories: [development]
+    icon: ollama
+    color: mauve
```

```suggestion
    color: text
```
Don't see any purple on the site, the color is meant to represent the brand color of the site.